Wangscape/tilegen/alpha/CalculatorMode.h

```diff
@@ -13,5 +15,7 @@ enum class CalculatorMode
     Dither
 };
 
+extern const std::map<std::string, CalculatorMode> calculatorModeByName;
```

Comment:
You need `#include <string>`. Clang's error messages for this are terribly unhelpful :(

Reply:
Eh... I was misled by a fact it builds on `gcc`. Headers seemed to be OK.